app/controllers/v1/clusters_controller.rb

```diff
@@ -1,32 +1,37 @@
 class V1::ClustersController < ApplicationController
-  skip_before_action :authorize_request
+  #skip_before_action :authorize_request
```

Comment:
Yes. When uncommented it skips checking the JWT tokens for the sake of testing the API manually without having to get new tokens every time.